extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -67,8 +69,16 @@ export class PodcastPlayer extends Component {
 		if ( ! trackData ) {
 			return;
 		}
+
 		this.setState( { currentTrack: track } );
 		this.setAudioSource( trackData.src );
+
+		// Read that we're loading the track and its description. This is dismissible via ctrl on VoiceOver.
+		speak(
+			`${ sprintf( __( 'Loading: %s', 'jetpack' ), trackData.title ) } ${ trackData.description }`,
```

Comment:
Let's use `_x()` to add translation context about the placeholder

Reply:
Please don't use `_x()`! Use a `translators:` comment.

Reply:
Removed `_x()` using `translators:` comment instead.

Reply:
@akirk I was not aware those worked outside of php, too. Neat! Should it still be in the line above the translation string?

Reply:
Yes, as adjacent as possible.